queries/gowork/highlights.scm

```diff
@@ -0,0 +1,14 @@
+[
+  "replace"
+  "go"
+  "use"
+] @keyword
+
+"=>" @operator
+
+(comment) @comment
```

Comment:
you could also add this to the injections (for example https://github.com/nvim-treesitter/nvim-treesitter/blob/296c28be5396da069e436f58bdaa541506f99fb5/queries/lua/injections.scm#L53)

Reply:
Okay took me a while to understand what this is supposed to do but I think I got it... so there's a "language" called "comment" and you want me to tell nvim-treesitter that any text within a `go.work`-file comment node should be treated as code in a different language, called "comment", so that for example words like TODO, FIXME, etc. would be highlighted?

Reply:
Force pushed, also added injections for gomod